src/ui/common/src/components/AnnouncementBanner/AnnouncementBanner.tsx

```diff
@@ -88,7 +88,9 @@ export const AnnouncementBanner: React.FC<AnnouncementBannerProps> = ({
       }
     }
 
-    fetchVersionNumber();
+    if (user.apiKey) {
```

Comment:
Pretty simple fix, this endpoint was being hit too many times before the apiKey was ready.